frontend/src/pages/ErrorsV2/ErrorInstance/AiErrorSuggestion.tsx

```diff
@@ -0,0 +1,193 @@
+import { NetworkStatus } from '@apollo/client'
+import { Button } from '@components/Button'
+import { useGetErrorResolutionSuggestionLazyQuery } from '@graph/hooks'
+import {
+	Box,
+	ButtonIcon,
+	IconSolidRefresh,
+	IconSolidSparkles,
+	IconSolidThumbDown,
+	IconSolidThumbUp,
+	Stack,
+	Text,
+} from '@highlight-run/ui'
+import { message } from 'antd'
+import clsx from 'clsx'
+import { useState } from 'react'
+import ReactMarkdown from 'react-markdown'
+
+import AiErrorSuggestionCard from '@/pages/ErrorsV2/ErrorInstance/AiErrorSuggestionCard'
+import analytics from '@/util/analytics'
+
+import * as styles from './AiErrorSuggestion.css'
+
+type Props = {
+	errorObjectId: string
+}
+
+export const AiErrorSuggestion = ({ errorObjectId }: Props) => {
+	const [
+		getErrorResolutionSuggestion,
+		{ data, error, loading, networkStatus, refetch },
+	] = useGetErrorResolutionSuggestionLazyQuery({
+		notifyOnNetworkStatusChange: true,
+	})
+	const [voted, setVoted] = useState(false)
+
+	return (
+		<Box
+			p="16"
+			borderRadius="8"
+			cssClass={clsx(styles.aiSuggestion, {
+				[styles.aiSuggestionPrompt]: !data?.error_resolution_suggestion,
+			})}
+		>
+			{data?.error_resolution_suggestion ? (
+				<Stack flexDirection="column">
+					<AiErrorSuggestionCard
+						content={
+							<ReactMarkdown
+								components={{
+									li: ({ children }) => <li>{children}</li>,
+								}}
+							>
+								{data.error_resolution_suggestion}
+							</ReactMarkdown>
+						}
+					/>
+					<AiErrorSuggestionCard
+						content={
+							<>
+								<Text>Was this helpful?</Text>
+								<Box display="flex">
+									<Stack
+										border="secondary"
+										borderRadius="8"
+										align="center"
+										direction="row"
+										gap="0"
+									>
+										<Box p="1">
+											<ButtonIcon
+												icon={<IconSolidThumbUp />}
+												kind="secondary"
+												emphasis="low"
+												size="small"
+												onClick={() => {
+													if (!voted) {
+														analytics.track(
+															'AiErrorSuggestionFeedback',
+															{ value: 1 },
+														)
+														console.log(
+															'AiErrorSuggestionFeedback',
+															1,
+														)
+														setVoted(true)
+														message.success(
+															`Thanks for your feedback!`,
```

Comment:
cool! could we also send the error object id as metadata about the tracked vote?